src/Manifolds.jl

```diff
@@ -1,5 +1,6 @@
 module Manifolds
 
+using LinearAlgebra: AbstractMatrix
```

Comment:
```suggestion
```
I suspect it's something automatically inserted by VSCode?

Reply:
Or a new code formatter? I deleted it once but might have missed it this time. But yes, this was automatically added.

Reply:
I don't think formatter does that, see https://github.com/julia-vscode/julia-vscode/issues/1925 .